cucumber/policy/features/support/policy_helpers.rb

```diff
@@ -32,68 +32,127 @@ def invoke status: nil, &block
   end
 
   def load_root_policy policy
-    conjur_api.load_policy("root", policy, method: Conjur::API::POLICY_METHOD_PUT)
+    resource('root').put(policy, header)
   end
 
   def update_root_policy policy
-    conjur_api.load_policy("root", policy, method: Conjur::API::POLICY_METHOD_PATCH)
+    resource('root').patch(policy, header)
   end
 
   def extend_root_policy policy
-    conjur_api.load_policy("root", policy, method: Conjur::API::POLICY_METHOD_POST)
+    resource('root').post(policy, header)
   end
 
-  def load_policy id, policy
-    conjur_api.load_policy(id, policy, method: Conjur::API::POLICY_METHOD_PUT)
+  def load_policy(id, policy)
+    resource(id).put(policy, header)
   end
 
-  def update_policy id, policy
-    conjur_api.load_policy(id, policy, method: Conjur::API::POLICY_METHOD_PATCH)
+  def update_policy(id, policy)
+    resource(id).patch(policy, header)
   end
 
-  def extend_policy id, policy
-    conjur_api.load_policy(id, policy, method: Conjur::API::POLICY_METHOD_POST)
+  def extend_policy(id, policy)
+    resource(id).post(policy, header)
   end
 
-  def make_full_id *tokens
-    super(tokens.join(":"))
+  def create_api_key role
+    login_resource.put(
+      "", Authorization: create_token_header, params: { role: role }
+    )
+  end
+
+  def admin_api_key
+    admin_resource.get
+  end
+
+  def get_login_token login, key
+    RestClient.post(
+      uri('authn', CGI.escape(login), 'authenticate'),
```

Comment:
I would move `CGI.escape` inside `uri`, so that it only exists in one place and client code doesn't have to worry about it.